exporter/operators/new_export_collection.py

```diff
@@ -73,6 +74,7 @@ def invoke(self, context, event):
             name_content.append(bpy.context.active_object.name)
         export_name = "_".join(name_content)
         self.filename = get_export_filename(export_name, self.export_type)
+        self.filter_glob = "*.glb;*.obj" if get_preferences().use_gltf else "*.fbx;*.obj"
```

Comment:
I wonder if we could add something to _constants.py_ like `get_filter_glob()` which would automatically build this from whatever values were set in the `EXPORT_FILE_TYPES` dictionary? It would be more future-proof and maybe less invasive.